docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -465,6 +466,11 @@ _References:_
 - [http://nginx.org/en/docs/hash.html](http://nginx.org/en/docs/hash.html)
 - [https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_headers_hash_bucket_size](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_headers_hash_bucket_size)
 
+## plugins
+
+Configures plugins to use placed in the directory /etc/nginx/lua/plugins. Every plugin has to have main.lua in the root.
+Every plugin has to bundle all of its dependencies. The execution order follows the definition.
+
```

Comment:
I'm going to add a README in https://github.com/kubernetes/ingress-nginx/tree/master/rootfs/etc/nginx/lua/plugins/ and also create example test for the hello_world example plugin to make it more complete so that users can refer to it.